src/DnsRootStack.ts

```diff
@@ -51,153 +49,9 @@ export class DnsRootStack extends cdk.Stack {
-      zone: this.cspNijmegenZone,
-      recordName: '_578630817267b977076c44d1065f484d',
-      domainName: 'a213f88cb03ee3378d4a61b0ee419340.6f8522989e4f5cdbaa87ab19f464e270.comodoca.com.',
-    });
-    new Route53.CnameRecord(this, 'webformulieren-validation-record-accp', { // Webformulieren accp - esb (auth.accp.csp-nijmegen.nl)
-      zone: this.cspNijmegenZone,
-      recordName: '_9098af4bf46f252af5e3a050f3393016',
-      domainName: '361c615f2fecf47aaaa7c5b74a0a1ebe.33a6de06e8378e1c06dd092f6da142e2.comodoca.com.',
-    });
-
-    new Route53.CnameRecord(this, 'tribe-brp-validation-record-accp', { // Tribe brp accp - esb (tribebrp.generiek-accp.csp-nijmegen.nl)
-      zone: this.cspNijmegenZone,
-      recordName: '_2db6a8de32f9fb49b42f88127092be93',
-      domainName: '7a38240e33f335e0b935367644423084.efec36361a757e70d540a291701c27fb.comodoca.com.',
-    });
-    new Route53.CnameRecord(this, 'tribe-brp-validation-record-prod', { // Tribe brp prod - esb (tribebrp.generiek-prod.csp-nijmegen.nl)
-      zone: this.cspNijmegenZone,
-      recordName: '_7b3cf064cfc7fdbd40a7c6677563c523',
-      domainName: '4594484c90b02f72b15f7a26954f1481.dc969c51f59d7bd58e55f09f19b6882f.comodoca.com.',
-    });
-
-    new Route53.CnameRecord(this, 'irma-issue-validation-record-accp', { // Irma issue accp - esb (irma-issue.accp.csp-nijmegen.nl)
-      zone: this.cspNijmegenZone,
-      recordName: '_82e96918e2e006569edd57a9bf2ca18a',
-      domainName: '1c25b15c30dea2869a5cf44da8cfd07d.b4a989acf7f156b70e69c6c20996feed.comodoca.com.',
-    });
-    new Route53.CnameRecord(this, 'irma-issue-validation-record-prod', { // Irma issue prod - esb (irma-issue.auth-prod.csp-nijmegen.nl)
-      zone: this.cspNijmegenZone,
-      recordName: '_8bcedded666ccd9b575ab5d78840b91a',
-      domainName: '2a8fbd419e077abf49863ab79387f155.5afde26b969373212b99b4a97eab728a.comodoca.com.',
-    });
-
-    /**
-     * Electronisch statusformulier
-     */
-    new Route53.CnameRecord(this, 'esf-validation-record-prod', {
-      zone: this.cspNijmegenZone,
-      recordName: '_e6afe3b30f34b29943c7a25a57a64fc3',
-      domainName: '8a517865032f100eb489a4cb779bea43.3de2af4d2845abf6a916c2f487d7cdd2.comodoca.com.',
-    });
-    new Route53.CnameRecord(this, 'esf-validation-record-accp', {
-      zone: this.cspNijmegenZone,
-      recordName: '_0dda4864990029d7b9def59785242089',
-      domainName: '4bd2d704cc5f72f39cf605cf24500bde.13f6be6ddcec328123708fd74b620d25.comodoca.com.',
-    });
-
-    /**
-     * DocPoc Sectigo
-     */
-    new Route53.CnameRecord(this, 'docpoc-validation-record-accp', { // DocPoc accp - esb (docpoc-accp.csp-nijmegen.nl)
-      zone: this.cspNijmegenZone,
-      recordName: '_C09AF98DC2FE54695443BF987E37C329',
-      domainName: '98DCDAD749B9B0C55544167D8CE5202C.2F575C50C0E49874844BC0A5A49CAD06.sectigo.com.',
-    });
-
-  }
-
-  setupMailRecords() {
```

Comment:
Is het de bedoeling dat deze ook verdwijnen?

Reply:
Ja, deze bestonden alleen in de oude lz nog. Zie https://github.com/GemeenteNijmegen/dns-management/pull/159/files/6420c03d5cdb4dd40548a38af7ab3d5653ffe821#diff-1ac6a279ffd399f65c07011ad4234ebed93d805dab750179f98981d0a32bfb63L56